go.mod

```diff
@@ -130,23 +109,23 @@ require (
 	gopkg.in/dancannon/gorethink.v3 v3.0.5 // indirect
 	gopkg.in/fatih/pool.v2 v2.0.0 // indirect
 	gopkg.in/gorethink/gorethink.v3 v3.0.5 // indirect
-	gopkg.in/inf.v0 v0.9.1 // indirect
 	gopkg.in/yaml.v2 v2.2.4
-	k8s.io/api v0.0.0-20190919035539-41700d9d0c5b
-	k8s.io/apimachinery v0.0.0-20190917163033-a891081239f5
-	k8s.io/cli-runtime v0.0.0-20190918162238-f783a3654da8
-	k8s.io/client-go v0.0.0-20190918160344-1fbdaa4c8d90
-	k8s.io/component-base v0.0.0-20190918040032-61bc4cc48c91
-	k8s.io/klog v0.4.0
-	k8s.io/kubectl v0.0.0-20190919041832-5bf870509147
-	k8s.io/utils v0.0.0-20190907131718-3d4f5b7dea0b // indirect
+	helm.sh/helm/v3 v3.0.2
+	k8s.io/api v0.0.0-20191016110408-35e52d86657a
+	k8s.io/apimachinery v0.0.0-20191004115801-a2eda9f80ab8
+	k8s.io/cli-runtime v0.0.0-20191016114015-74ad18325ed5
+	k8s.io/client-go v0.0.0-20191016111102-bec269661e48
+	k8s.io/component-base v0.0.0-20191016111319-039242c015a9
+	k8s.io/klog v1.0.0
+	k8s.io/kubectl v0.0.0-20191016120415-2ed914427d51
 	sigs.k8s.io/kustomize v2.0.3+incompatible
 	sigs.k8s.io/yaml v1.1.0
 	vbom.ml/util v0.0.0-20180919145318-efcd4e0f9787 // indirect
 )
 
 replace (
 	github.com/Azure/go-autorest => github.com/Azure/go-autorest v12.4.3+incompatible
+	github.com/docker/docker => github.com/moby/moby v0.7.3-0.20190826074503-38ab9da00309
```

Comment:
i don't think this change is right? and it breaks things when you try to import the package. if we want to depend on this version of docker, shouldn't that be in the stanza above, rather than overriding it here?